fontir/src/ir.rs

```diff
@@ -948,6 +948,27 @@ fn is_ribbi(style_name: &str) -> bool {
 }
 
 impl NameBuilder {
+    /// If drop_style_suffix remove trailing Regular, Bold, or Italic, e.g. convert Family Regular to just Family
+    pub fn make_family_name(
+        family: &str,
+        subfamily: &str,
+        separator: &str,
```

Comment:
is `separator` ever not just `" "`? (I did a quick search and didn't see a counter example..)

Reply:
Oh good catch, maybe I ended up never using anything but space. Will fix.